index.html

```diff
@@ -3190,7 +3190,9 @@ <h2>Definition of Roles</h2>
 			<rdef>form</rdef>
 			<div class="role-description">
 				<p>A <rref>landmark</rref> region that contains a collection of items and objects that, as a whole, combine to create a form. See related <rref>search</rref>.</p>
-				<p>A form may be a mix of host language form controls, scripted controls, and hyperlinks. Authors are reminded to use native host language semantics to create form controls, whenever possible. For search facilities, authors SHOULD use the <rref>search</rref> role and not the generic <code>form</code> role. Authors SHOULD provide a visible label for the form referenced with <pref>aria-labelledby</pref>. If an author uses a script to submit a form based on a user action that would otherwise not trigger an <code>onsubmit</code> event (for example, a form submission triggered by the user changing a form element's value), the author SHOULD provide the user with advance notification of the behavior.</p>
+				<p>A form may be a mix of host language form controls, scripted controls, and hyperlinks. Authors are reminded to use native host language semantics to create form controls, whenever possible. For search facilities, authors SHOULD use the <rref>search</rref> role and not the generic <code>form</code> role.</p>
+				<p>Authors MUST give each element with role <code>form</code> a brief label that describes the purpose of the content in the form. Authors SHOULD reference a visible label with <pref>aria-labelledby</pref> if a visible label is present. Authors SHOULD include the label inside of a heading whenever possible. The heading MAY be an instance of the standard host language heading element or an instance of an element with role <rref>heading</rref>.</p> 
```

Comment:
Just out of curiosity, why is it desirable for authors to include the label inside of a heading? (I see that the same language is included for `region`; I'm questioning its applicability to `form`.)

Reply:
if the WebAIM survey results are anything to go by, the reporting that navigating by landmarks has been declining year to year, while navigating by headings remains a consistently preferred method of navigation, would be one reason to call this out.

Reply:
I had the same thoughts. Both of these seem more appropriate for APG rather than for the spec.
I think the benefit is in giving people using different navigation methods in screen readers the ability to get at the content. I don't think it is a bad pattern but I do question its relevance for the spec itself.

Reply:
Ok, I can go either way. (i.e. make a decision, mark this convo as resolved, and land). Thanks for the thoughts!